extensions/auto-html.js

```diff
@@ -1,5 +1,6 @@
 var linkRegex = /((https?:\/\/)([\da-z\.-]+)\.([a-z\.]{2,6})([\/\w \.-]*)*\/?)/gi;
 var imageLinkRegex = /(<a[^>]+>)([^<]+\.(png|gif|jpg))<\/a>/gi;
+var roomMeRegex = /#([A-Za-z0-9_-]+)(?![A-Za-z0-9_\]-])/g;
```

Comment:
This will break links like ```http://page.com/steve.html#hooray``` maybe we should update it so the link has to be preceded by a space or at the start of the message.